src/panels/lovelace/hui-root.ts

```diff
@@ -1013,8 +1013,6 @@ class HUIRoot extends LitElement {
           padding-inline-start: env(safe-area-inset-left);
           padding-inline-end: env(safe-area-inset-right);
           padding-bottom: env(safe-area-inset-bottom);
-        }
-        hui-view {
```

Comment:
This should be reverted I think.

The issue is that the background can be set with a theme variable `--lovelace-background` that can set on the view or with a config variable in YAML. Putting the background option a level higher means that the view theme can not set the background of the view, because it can not target anything above itself.

You can set the background on both `#view` and `hui-view`, but that will not give the best experience either.

Reply:
Or, a but ugly... https://github.com/home-assistant/frontend/pull/20683

Reply:
What config was needed to expose the issue? I'm setting the background using a theme and that works fine:
```
  modes:
    light:
      background-image: "center / cover no-repeat fixed url('https://s3-alpha-sig.figma.com/img/37a7/bb5c/3cce1cb84cadd7abd5c805a1c1e605ec?Expires=1715558400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=LSJpRSeVxMzynEfxp4gddeC5d9JBtjnjaxY7wxxrTkzoJAOvAYe~o3u~8hpu0dOFpepDNvvM8SiFhn1f892NGoNCDx-BoJWfxp0jzSFYz2riMUY0RLZ-rl8W7bfluvmBzzpOliTGtMN2~roBMxsDuBvpjyz9u8E3YN9S6PQWjP~8sLLS0HDmf7m8qaaXV2aTgc06nPrB8vmpxFmRMkS-UHEmMKD2Rlj-LTp~ewRR4RIxntITPtAAPuF5NFh99quVZNHbmOUeBOhr5fJUVeVmdnXJtAxCgYRvQAeAzplBLSM8kqvSgpZJOggdmnW5V-y7plhj55KGWKgurPT9~0C03w__')"
    dark:
      background-image: "center / cover no-repeat fixed url('https://s3-alpha-sig.figma.com/img/d3bc/29e6/8ffb48c35f93bda494c79e9fea86be94?Expires=1715558400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=T-Jak5r4mcb3R0h6vmERm5k1c0pTaSIWOJH-vk5TpYgW1nRsj-WqPwNfAa844Uvevt1wdTCV-P8nCCF8VUB5Ah9SKevfMsTWRPgsthItw6VcuOTvaUY8cyCO9RcgPzlaxUM6hxw7Jcen2i8rmdwFukGVFWeVnEr5fxET1K1a8OspxIl~jsR1OuxgG~yxcyngaucizsQbjdLpBWPX2hyu1wznaoAPq5DaVGM3giUZXXo6nY34jjclGhQrYm3TWR3Ktjnl6CWXdqfBT9dMo~mqaitCnetdpcBxSErB4QJx9-DtTfHodlNDilBHqPB2j5qwRlhzX05Hk9fqEp4yeVnYdw__')"
  lovelace-background: var(--background-image)
```